frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx

```diff
@@ -26,9 +32,66 @@ const OrderInfo = ({ orderData }) => {
     return quantity;
   }, [cartItems]);
 
-  // const sendOrder = () => {
-  //   console.log(cartItems, orderData);
-  // };
+  const sendOrder = async () => {
+    const orderItems = cartItems.map((item) => {
+      if (item.orderItemsCompositions) {
+        const bouquetItems = item.orderItemsCompositions.map(
```

Comment:
Would it be smarter to use this structure when passing data to the cart?

Reply:
Could you please explain what you mean?